R/summarize_weighted_production.R

```diff
@@ -61,7 +61,9 @@ summarize_weighted_production_ <- function(data, ..., use_credit_limit = FALSE,
 
   old_groups <- dplyr::groups(data)
 
-  crucial <- c("production", "sector_ald", "year", "technology")
+  data <- rename_and_warn_ald_names(data)
```

Comment:
`rename_and_warn_ald_names` not `rename_and_warn_abcd_names`?

Reply:
Ya, i mean it "checks if there are names that contain `ald`, and if there are, it warns and renames them". I think the name is valid.